fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/physical/PhysicalOperator.java

```diff
@@ -56,8 +56,4 @@ public ColumnRefSet getUsedColumns() {
 
         return result;
     }
-
-    public abstract boolean equals(Object o);
```

Comment:
keep may be better,  others may forget implents these method when extends this class

Reply:
equal has been implemented in the parent class according to predicate and limit, and the subclass needs to implement its own function according to its own class attributes